src/UserContext.ts

```diff
@@ -27,9 +28,17 @@ const defaultAcl = {
   },
 };
 
-const defaultContext = {
+const defaultDowntime = {
+  default_duration: 3600,
```

Comment:
Currently, default duration is 2 hours
```suggestion
  default_duration: 7200,
```